Dockerfile

```diff
@@ -392,6 +368,22 @@ RUN boot -u
 
 USER root
 
+ADD lib/dependencies /php/dependencies/
+ADD lib/php/5.6 /php/5.6
+ADD lib/php/7.2 /php/7.2
+ADD lib/php/7.4 /php/7.4
+
+#RUN apt-get -y update && apt-get -y install libpcre3
```

Comment:
leftover code?

Reply:
Ish, after I marked this ready for review, I spot some problems with iptables in buildbot as mentioned in the slack channel and also in [issue](https://github.com/netlify/buildbot/issues/1419), but didn't get to move the PR back to draft.